src/maps/Map.js

```diff
@@ -196,4 +195,7 @@ export default class Map {
         m[15] = 1;
         return m;
     }
+
+    triggerRepaint () {
```

Comment:
Should we add here `window.requestAnimationFrame(this._update.bind(this));` ? Or add a comment explaining we need it to avoid problems when calling `this.map.triggerRepaint();` in the Layer is not a mgl map? (not a big fan of comments, but leaving it empty looks weird to me). Just a minor comment.

Reply:
Well, as you said, this needs to do something (the code you posted seems good actually) to make animations work. The reason I left it unimplemented is that I think that `Map` already didn't work with animation (so it wasn't needed to implement it).

I agree with you in that it is weird. But I think the real problem is that we are not clarifying the whole purpose of the stand-alone `Map`. We are only using it for some tests (and I think it wouldn't be very problematic to replace it by MGL in every test, we would need to check times though), we are exposing it on the `carto.` namespace without an underscore (although it is not exposed in the public documentation). On the other hand, it is good to have something that forces us to don't couple CVL to MGL.

Reply:
Yes, I agree that is good to have it, and it is really useful for testing. I think it would be great to explain what `Map` is, but I am not sure if this is necessary at this point.